proposals/typography-components.md

```diff
@@ -0,0 +1,64 @@
+# Typography package
+
+This proposal is to create a `baseui/typography` package containing the typography components. These components codify the [BaseUI typography standards](https://www.figma.com/file/L7MSVAit4clxkfGE22NMX86Q/%E2%9D%96-Base-UI?node-id=19590%3A824) for H1 .. H6, Display, Caption, CaptionLabel, Label1, Label2, P1, P2,
+
+## Motivation
+
+Form interfaces, as well as informational pages, in Web apps have a text blocks. BaseUI has a set of standards for these text elements. Designers use them in their designs. This proposal enables developers to use these elements just like other BaseUI components.
+
+## Usage
+
+```javascript
+import * as React from 'react';
+import {H1, P1} from 'baseui/typography';
+
+export default () => (
+  <React.Fragment>
+    <H1>BaseUI Heading/H1 Title</H1>
+    <P1>We ignite opportunity by setting the world in motion</P1>
+  </React.Fragment>
+);
+```
+
+## Exports
+
+* `Display`
+* `Caption`
+* `CaptionLabel`
+* `H1`
+* `H2`
+* `H3`
+* `H4`
+* `H5`
+* `H6`
+* `Label1`
+* `Label2`
+* `P1`
+* `P2`
+
+## `Display`, `CaptionLabel`, `Label1`, `Label2` API
+
+Same as [<div> MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/div)
+
+## `H1`, `H2`, `H3`, `H4`, `H5`, `H6` API
+
+Same as [<h1>–<h6> MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements)
+
+## `Caption` API
```

Comment:
Based on our discussion: 
- Introduce `Caption1` and `Caption2` instead of `Caption` and `CaptionLabel`. Those accordingly refer to `Caption1 === Caption`, `Caption2 === CaptionLabel`
- Don't use `<caption>` tag but `div` or `p`, same as for Display and Label